core/src/com/mygdx/game/models/Battle.java

```diff
@@ -0,0 +1,26 @@
+package com.mygdx.game.models;
+
+
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+
+public class Battle {
+    Set<Integer> playerIds;
+    Map<Integer, List<BasicUnit>> units;
+    private Map<Integer, Integer> strengths;
```

Comment:
Did we still want to have another stat which determined which unit took the tile that was different that strength